src/test/java/athena/StorageTest.java

```diff
@@ -0,0 +1,53 @@
+package athena;
+
+import athena.task.Task;
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class StorageTest {
+
+    @Test
+    void saveTaskListData_noPreviousSave_createSaveFile() {
+        Ui ui = new Ui();
+        TaskList taskList = TestSetup.getTestTaskList();
+        Storage storage = new Storage("src/test/java/athena/loadTask.csv", ui);
+        storage.saveTaskListData(taskList);
+        assertTrue(areFilesSame("src/test/java/athena/loadTask.csv", "src/test/java/athena/TaskAnswer.csv"));
+    }
+
+    private boolean areFilesSame(String file1, String file2) {
+        BufferedReader answer = null;
+        try {
+            answer = new BufferedReader(new FileReader(file1));
+            BufferedReader attempt = new BufferedReader(new FileReader(file2));
+            String answerLine = answer.readLine();
+            String attemptLine = attempt.readLine();
+            while ((answerLine != null) || (attemptLine != null)) {
+                if (!answerLine.contentEquals(attemptLine)) {
+                    return false;
+                } else {
+                    answerLine = answer.readLine();
+                    attemptLine = attempt.readLine();
+                }
```

Comment:
`else` is not needed since this will always be reached when the `if` condition is not satisfied.